app/views/layouts/_devise_shared.html.erb

```diff
@@ -14,8 +14,11 @@
   <link href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.11.2/css/all.min.css" rel="stylesheet" />
   <link href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/v4-shims.css" rel="stylesheet" />
 
-  <%= raw fullstory_script(current_user: current_user) if Rails.env.production? %>
   <%= javascript_importmap_tags %>
+  <%=
+    # DO NOT MOVE THIS ABOVE THE IMPORT MAPS
```

Comment:
Please put this warning on the import maps instead -- "DO NOT LOAD ANY JAVASCRIPT BEFORE THIS!!!"